hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -102,6 +102,12 @@
     FRONTEND_GRACEFUL_SHUTDOWN_INITIAL_WAIT_MS("frontend.graceful.shutdown.initial.wait.ms", 10000),
     FRONTEND_HTTP2_ENABLED("frontend.http2.enabled", false),
     FRONTEND_FORCE_TOPIC_MAX_MESSAGE_SIZE("frontend.force.topic.max.message.size", false),
+    FRONTEND_THROUGHPUT_TYPE("frontend.throughput.type", "unlimited"),
+    FRONTEND_THROUGHPUT_MAX("frontend.throughput.max", Long.MAX_VALUE),
```

Comment:
If these 5 parameters apply to `dynamic` implementation only, then let's put them in a proper namespace, e.g. `frontend.throughput.dynamic.max`.